server/index.js

```diff
@@ -33,18 +33,10 @@ export default class Server {
 
   getRequestHandler () {
     return (req, res, parsedUrl) => {
-      if (!parsedUrl) {
+      if (!parsedUrl || parsedUrl && !parsedUrl.query) {
```

Comment:
@sedubois since it's an `or` both sides of the expression are executed. In this case when `parsedUrl` is `undefined` it will throw an `query` is not defined error.